data/json/items/melee/bludgeons.json

```
@@ -1053,9 +1050,9 @@
     "techniques": [ "WBLOCK_1", "BRUTAL", "SWEEP" ],
     "flags": [ "DURABLE_MELEE", "SPEAR", "NONCONDUCTIVE", "BELT_CLIP" ],
     "volume": "1250 ml",
-    "bashing": 20,
-    "cutting": 20,
+    "bashing": 22,
+    "cutting": 23,
     "price": 16000,
-    "qualities": [ [ "HAMMER", 1 ] ]
+    "qualities": [ [  "HAMMER", 1 ] ]
```

```suggestion
    "qualities": [ [ "HAMMER", 1 ] ]
```
JSON validator doesn't like doublespace

---

Thank you.